### ISSUE TITLE
After accepting takeback both players left waiting for opponent to move

### ISSUE THREAD

**Issue by csaavedra** [veloce/lichobile#1863]
First reported this here: https://lichess.org/forum/lichess-feedback/bug-after-accepting-takeback

As described there, this happened several times with the same opponent. After accepting a takeback request, their clock would start running and I would be left several minutes waiting for their move. After seeing they were about to run out of time I asked them why they were not moving and they laughed and replied that it was my turn. However I couldn't move, as their clock was running.

The first time it happened we decided to agree on a draw because neither of us knew what to do. Here is the drawn game: https://lichess.org/MUlW0vSf14PR

We then proceeded to rematch and play a couple more games. In one of them they asked for a takeback, and after accepting it, the same happened. This time when I noticed their clock started running I went to lichess.org on my phone's browser, and then I could see it was my turn. Moving in the browser re-sync'd the app, but then after another takeback the same happened again. I lost a lot of time doing this during this blitz game and ended up losing due to all the distraction.

Long story short, looks like some synchronization issue. I don't know if it's really a problem in the app, but some kind of periodic synchronization during games can be useful to avoid this.

**Comment by jas14** [comment]
Thanks for the report @csaavedra - I'm going to mark this as a duplicate of #1640 to consolidate. please give that issue a thumbs-up!

**Comment by csaavedra** [comment]
Sorry, didn't check thoroughly!
